CONTRIBUTING.md

```diff
@@ -93,3 +93,35 @@ For merging, you should:
        Travis to run the tests for each change you add in the pull request.
        Because testing here will delay tests by other developers, please ensure
        that the code passes all tests on your local system first.
+
+Project Style Guidelines
+------------------------
+
+In general, pysat follows PEP8 and numpydoc guidelines.  Pytest runs the unit
+ and integration tests, flake8 checks for style, and sphinx-build performs
+documentation tests.  However, there are certain additional style elements that
+have been settled on to ensure the project maintains a consistent coding style.
+These include:
+
+* Line breaks should occur before a binary operator (ignoring flake8 W503)
+* Combine long strings using `join`
+* Preferably break long lines on open parentheses rather than using `\`
```

Comment:
Since 79 is rapidly becoming the default, we should also call out the line limit
```suggestion
* Preferably break long lines on open parentheses rather than using `\`
* Use no more than 80 characters per line
```

Reply:
I'll wait until you make all your changes before committing them to reduce the amount of CI runs.

Reply:
That's it from me.  The other special things we ignore will probably be removed down the line, so we can revisit them later.